### ISSUE TITLE
This code is for which paper

### ISSUE THREAD

**Issue by maidoudouyang** [mridulv/action_recognition#1]
hello,i have a question about this code.I do not know which paper for this code.

**Comment by mridulv** [comment]
It has been a year since I did this project. So I am not able to recall
which paper I had implemented.
Sorry for the inconvenience.

**Comment by maidoudouyang** [comment]
That's all right!Could you tell me some key words or information about the paper? Thank you for your help

**Comment by ccdelgado** [comment]
The dropbox link are eliminated!

**Comment by pioneer911** [comment]
The dropbox link are eliminated,So I don't the structure and content of the DATA folder ,Can you resend the dropbox link?

**Comment by pioneer911** [comment]
@ccdelgado Do you figure out the content and structure of the DATA ?Thanks a lot.

**Comment by pioneer911** [comment]
@maidoudouyang  The dropbox link are eliminated.Can you tell me the content and structure of the DATA and other folders ?Thanks a lot.

**Comment by harrychen9999** [comment]
The dropbox link are eliminated,So I don't the structure and content of the DATA folder ,Can you resend the dropbox link?

**Comment by LOKESH2018** [comment]
@maidoudouyang Can you send the contents of dropbox contents or structure of it.Thanks
